<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>colorSelection</class>
 <widget class="QWidget" name="colorSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="colorSelect">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>251</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>Black</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>White</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Red</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Orange</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Yellow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Green</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Blue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Indigo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Violet</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>340</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>340</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLabel" name="CurrentColorText">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Current Color</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CurrentColorPreview">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="NewColorPreview">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="NewColorText">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>New Color</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="rSlider">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>45</y>
     <width>225</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="gSlider">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>115</y>
     <width>225</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="bSlider">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>185</y>
     <width>225</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="aSlider">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>255</y>
     <width>225</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="rText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>R</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="gText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>G</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="bText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>B</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="aText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="rSpinBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="gSpinBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="bSpinBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="aSpinBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>255</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>colorSelection</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>aSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>aSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>bSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>bSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>gSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>gSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
